tools/bin/hawqregister

```diff
@@ -847,6 +819,33 @@ class HawqRegister(object):
                 self._modify_metadata('insert')
         logger.info('Hawq Register Succeed.')
 
+class HawqRegister_Partition(HawqRegister):
```

Comment:
Class name in this file is like this format:AaaBbbCcc, so HawqRegisterPartition is better here.